src/locale/locales/de/messages.po

```diff
@@ -2162,217 +2087,212 @@ msgstr "Diese Warnung ist nur für Beiträge mit angehängten Medien verfügbar.
 msgstr "Verwendet von:"
 
 #: src/view/com/auth/create/Step3.tsx:38
 msgid "User handle"
-msgstr "Benutzerhandgriff"
+msgstr "Benutzerhandle"
 
 #: src/view/screens/Lists.tsx:58
 msgid "User Lists"
-msgstr "Benutzer Liste"
+msgstr "Benutzerlisten"
 
 #: src/view/com/auth/login/LoginForm.tsx:170
 #: src/view/com/auth/login/LoginForm.tsx:187
 msgid "Username or email address"
-msgstr "Benutzername oder E-Mail Adresse"
+msgstr "Benutzername oder E-Mail-Adresse"
 
 #: src/view/screens/ProfileList.tsx:738
 msgid "Users"
 msgstr "Benutzer"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:143
 msgid "users followed by <0/>"
-msgstr "Benutzer gefolgt von <0/>"
+msgstr "Nutzer gefolgt von <0/>"
 
 #: src/view/com/modals/Threadgate.tsx:106
 msgid "Users in \"{0}\""
-msgstr "Benutzer in \"{0}\""
+msgstr ""
 
 #: src/view/screens/Settings.tsx:769
 msgid "Verify email"
-msgstr "E-Mail überprüfen"
+msgstr "E-Mail bestätigen"
 
 #: src/view/screens/Settings.tsx:794
 msgid "Verify my email"
-msgstr "E-Mail überprüfen"
+msgstr "Meine E-Mail bestätigen"
 
 #: src/view/screens/Settings.tsx:803
 msgid "Verify My Email"
-msgstr "E-Mail überprüfen"
+msgstr "Meine E-Mail bestätigen"
 
 #: src/view/com/modals/ChangeEmail.tsx:205
 #: src/view/com/modals/ChangeEmail.tsx:207
 msgid "Verify New Email"
-msgstr "Neuer E-Mail überprüfen"
+msgstr "Neue E-Mail bestätigen"
 
 #: src/view/screens/Log.tsx:52
 msgid "View debug entry"
 msgstr "Debug-Eintrag anzeigen"
 
 #: src/view/com/profile/ProfileSubpageHeader.tsx:128
 msgid "View the avatar"
-msgstr "Avatar sehen"
+msgstr "Avatar ansehen"
 
 #: src/view/com/modals/LinkWarning.tsx:73
 msgid "Visit Site"
-msgstr "Seite besuchen"
+msgstr "Seite ansehen"
 
 #: src/view/com/auth/create/CreateAccount.tsx:121
 msgid "We're so excited to have you join us!"
-msgstr "Wir freuen uns, Sie bei uns begrüßen zu dürfen!"
+msgstr "Wir freuen uns sehr, dass du dabei bist!"
 
 #: src/view/screens/Search/Search.tsx:236
 msgid "We're sorry, but your search could not be completed. Please try again in a few minutes."
-msgstr "Es tut uns leid, aber Ihre Suche konnte nicht abgeschlossen werden. Bitte versuchen Sie es in ein paar Minuten erneut."
+msgstr "Es tut uns leid, aber deine Suche konnte nicht abgeschlossen werden. Bitte versuche es in ein paar Minuten erneut."
 
 #: src/view/screens/NotFound.tsx:48
 msgid "We're sorry! We can't find the page you were looking for."
-msgstr "Es tut uns leid! Wir können die Seite, die Sie suchen, nicht finden."
+msgstr "Es tut uns leid! Wir können die Seite, nach der du gesucht hast, nicht finden."
 
 #: src/view/com/auth/onboarding/WelcomeMobile.tsx:46
 msgid "Welcome to <0>Bluesky</0>"
 msgstr "Willkommen bei <0>Bluesky</0>"
 
 #: src/view/com/modals/report/Modal.tsx:169
 msgid "What is the issue with this {collectionName}?"
-msgstr "Was ist das Problem mit dieser {collectionName}?"
-
-#: src/view/com/auth/SplashScreen.tsx:34
-#~ msgid "What's next?"
-#~ msgstr "Was kommt als Nächstes?"
+msgstr ""
```

Comment:
`"Was ist das {collectionName}-Problem?"` would not be always be perfect grammar, but possible